src/bridge/BitcoinTx.sol

```diff
@@ -128,6 +128,17 @@ library BitcoinTx {
         bytes bitcoinHeaders;
     }
 
+    /// @notice Represents info about an unspent transaction output.
+    struct UTXO {
+        /// @notice Hash of the transaction the output belongs to.
+        /// @dev Byte order corresponds to the Bitcoin internal byte order.
+        bytes32 txHash;
+        /// @notice Index of the transaction output (0-indexed).
+        uint32 txOutputIndex;
+        /// @notice Value of the transaction output.
+        uint64 txOutputValue;
```

Comment:
we don't really need this - we only need the outpoint (hash+index). But it doesn't matter a ton I guess

Reply:
imported from tbtc [here](https://github.com/keep-network/tbtc-v2/blob/cadead9ecd6005325ace4d64288c20733b058352/solidity/contracts/bridge/BitcoinTx.sol#L146-L158)